src/main/resources/migrations.xml

```diff
@@ -2711,6 +2711,13 @@
 		</rollback>
 	</changeSet>
 
+	<changeSet id="delete-peoplePositions-without-position-and-person" author="sahindoganlutfi" dbms="mssql,postgresql">
+		<comment>Cleanup the NULL of the personUuid and positionUuid columns of the peoplePositions table.</comment>
+		<sql>
+			DELETE FROM "peoplePositions" WHERE "personUuid" IS NULL and "positionUuid" IS NULL;
+		</sql>
+	</changeSet>
+
```

Comment:
Please add new changesets at the bottom of `migrations.xml` so they appear in the master branch in a meaningful order. Also, if a migration is not database-specific, leave out the `dbms="mssql,postgresql"`.

Reply:
Changeset moved and fixed.